tests/sentry/workflow_engine/migration_helpers/test_issue_alert_migration.py

```diff
@@ -373,6 +375,17 @@ def test_run__every_event_condition__all(self):
         dc = DataCondition.objects.get(type=Condition.REGRESSION_EVENT)
         assert dc.condition_group.logic_type == DataConditionGroup.Type.ALL
 
+    def test_run__lock(self):
+        lock = locks.get(
+            f"workflow-engine-project-error-detector:{self.project.id}",
+            duration=10,
+            name="workflow_engine_issue_alert",
+        )
+        lock.acquire()
+
+        with pytest.raises(UnableToAcquireLock):
+            IssueAlertMigrator(self.issue_alert, self.user.id).run()
```

Comment:
do we want this exception to bubble up and create sentry issues / ping in the channel (or should we handle it in code)?

Reply:
oh hm i wouldn't want to create a sentry error but i would want it to stop the migration 🤔